examples/queryReferenceCatalog.py

```diff
@@ -36,8 +36,8 @@ def main():
     indexid = ids[0]
 
     idName = 'id'
-    X = solver.getCatalogue(ra * afwGeom.degrees, dec * afwGeom.degrees,
-                            radius * afwGeom.degrees, '', idName, indexid)
+    X = solver.getCatalogue(ra * geom.degrees, dec * geom.degrees,
```

Comment:
You might also remove the spaces around the `*` operators, but I recognize that that would make the style inconsistent with the rest of the file.